cmd/process-agent/collector_api_test.go

```diff
@@ -582,6 +582,7 @@ func newMockEndpoint(t *testing.T, config *endpointConfig) *mockEndpoint {
 	collectorMux := http.NewServeMux()
 	collectorMux.HandleFunc("/api/v1/validate", m.handleValidate)
 	collectorMux.HandleFunc("/api/v1/collector", m.handle)
```

Comment:
Does that still make sense to keep that one in the unit test while you're not using it anymore? Just a confidence check, I did not dig into the unit tests

Reply:
Yes, thank you, it is still used.